src/rnp/fficli.cpp

```diff
@@ -1360,7 +1360,7 @@ rnp_cfg_set_ks_info(rnp_cfg_t *cfg)
 
     /* getting path to keyrings. If it is specified by user in 'homedir' param then it is
      * considered as the final path */
-    if (rnp_cfg_getint_default(cfg, CFG_KEYSTORE_DISABLED, 0)) {
+    if (rnp_cfg_getint_default(cfg, CFG_KEYSTORE_DISABLED, false)) {
```

Comment:
rnp_cfg_getbool() will work in the same way.

Reply:
Exactly! Thanks)